library/unwind/src/lib.rs

```diff
@@ -68,5 +91,5 @@ extern "C" {}
 extern "C" {}
 
 #[cfg(all(target_vendor = "fortanix", target_env = "sgx"))]
-#[link(name = "unwind", kind = "static-nobundle")]
+#[link(name = "unwind", kind = "static")]
```

Comment:
Every use of `static` instead of `static-nobundle` is a potential future headache, *especially* if it's used under link-time `cfg`.
If https://github.com/rust-lang/rust/issues/72274 gets fully implemented and the "unbundling" logic (*) gets removed from the compiler, then `static` and `cfg` will no longer be compatible.

(*) Even now the unbundling is not implemented to work in a general case, only to the bare minimum extent required for `libc` to work (apparently it makes `libunwind` work too).

Reply:
I plan to implement this in another PR. I hope this PR should not break anything.